lib/zoi/types/datetime.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Zoi.Types.Datetime do
+  @moduledoc false
+  use Zoi.Type.Def, fields: [:format]
```

Comment:
`format` field is not being used anymore

Reply:
[address issues](https://github.com/phcurado/zoi/pull/11/commits/fe4578e2f85642ba364992f672af9fd79b79b581)